Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -1157,7 +1157,8 @@ private void ReconcilePointers()
                 {
                     if (pointerData.Pointer is IMixedRealityNearPointer)
```

Comment:
I think this should be able to be updated to the following, and you can remove the `var nearPointer = (IMixedRealityNearPointer)pointerData.Pointer;` line, so we aren't checking type / casting twice.


```suggestion
                    if (pointerData.Pointer is IMixedRealityNearPointer nearPointer)
```